spannerlib/span.py

```diff
@@ -1,3 +1,5 @@
+"""Span class and how to interface it with pandas"""
```

Comment:
how did you change this without changing the notebooks?

Reply:
I think the nbdev_export did that, maybe nbdev version thing?

Reply:
yes